src/rounded_rect_radii.rs

```diff
@@ -0,0 +1,104 @@
+// Copyright 2021 The kurbo Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! A description of the radii for each corner of a rounded rectangle.
+
+use std::convert::From;
+
+/// Radii for each corner of a rounded rectangle.
+#[derive(Clone, Copy, Default, Debug, PartialEq)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+pub struct RoundedRectRadii {
+    /// The radius of the top-left corner.
```

Comment:
There's a minor pedantic point here. I believe the use of the word "top" here is essentially assuming a y-down coordinate space, but kurbo is also supposed to work in y-up (mathematics) coordinate spaces. I don't think we have to do any kind of awkward renaming, but I do think it would be useful to at least add a comment to the struct explaining the story (I think "top" is always the smaller coordinate, so top and bottom are inverted in a y-up space).

Reply:
Okay, I've documented this at the top of `RoundedRectRadii`.